terraform/functions.tf

```
@@ -308,7 +308,7 @@ module "lambda_function-graphql" {
         "s3:HeadObject",
       ]
       resources = [
-        # Path: treasuryreports/{organization_id}/{reporting_period_id}/{filename}.csv
+        # Path: treasuryreports{organization_id}/{reporting_period_id}/{filename}.csv
```

We discussed already but for posterity: looks like this change was unintentional.
```suggestion
        # Path: treasuryreports/{organization_id}/{reporting_period_id}/{filename}.csv
```